src/oc/audit.cpp

```diff
@@ -210,7 +210,7 @@ static void xopen_audit(void) {
     strcpy(buf, "rm ");
     bp = buf + strlen(buf);
     /* get the temporary file name */
-    nrn_assert(fgets(bp, 200, retrieve_audit.pipe));
+    nrn_assert(fgets(bp, 200 - 3, retrieve_audit.pipe));
```

Comment:
so this magic number 3 is `strlen("rm ")` ?

Reply:
yes